File: src/main/java/de/scandio/e4/testpackages/virtualusers/VirtualUserB.java
package de.scandio.e4.testpackages.virtualusers;

import de.scandio.e4.worker.interfaces.Scenario;
import de.scandio.e4.worker.interfaces.VirtualUser;

import java.util.List;

public class VirtualUserB implements VirtualUser {
    @Override
    public List<Class<Scenario>> getScenarios() {
        return null;
    }
}

File: src/main/kotlin/de/scandio/e4/testpackages/virtualusers/VirtualUserB.kt
package de.scandio.e4.testpackages.virtualusers

import de.scandio.e4.worker.interfaces.Scenario
import de.scandio.e4.worker.interfaces.VirtualUser

class VirtualUserB : VirtualUser {
    override fun getScenarios(): List<Class<Scenario>> {
        val list = arrayListOf<Class<Scenario>>()
        return list
    }
}